modules/accredited_representative_portal/app/policies/accredited_representative_portal/saved_claim_claimant_representative_policy.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class SavedClaimClaimantRepresentativePolicy < ApplicationPolicy
+    def index?
+      authorize
+    end
+
+    private
+
+    def authorize
+      @user.user_account.active_power_of_attorney_holders.size.positive?
```

Comment:
Copying [this comment](https://github.com/department-of-veterans-affairs/vets-api/pull/22646#discussion_r2140833627) over from @patrick-brown-oddball's work on the write side about which core facts about the user should play into authorization:

In a follow-up, we'll have to switch this component of the authorization logic.

`active_power_of_attorney_holders` is the subset of the representative's power of attorney holders that have their "digital POA request" flag turned on. We do not have this constraint for using our claim submission features.

Instead we'd want to expose and leverage `power_of_attorney_holders`.